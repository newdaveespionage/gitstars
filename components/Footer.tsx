import { colors } from '../theme'

const Footer = (): JSX.Element => (
  <footer>
    <a
      href="https://github.com/newdaveespionage/gitstars"
      target="_blank"
      rel="noopener noreferrer"
    >
      Created by Dave Espionage
    </a>
    <style jsx>{`
      footer {
        width: 100%;
        height: 100px;
        border-top: 1px solid ${colors.border};
        display: flex;
        justify-content: center;
        align-items: center;
      }

      footer a {
        display: flex;
        justify-content: center;
        align-items: center;
      }
    `}</style>
  </footer>
)

export default Footer
