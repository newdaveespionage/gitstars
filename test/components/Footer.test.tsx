import React from 'react'
import { render } from '../testUtils'
import Footer from '../../components/Footer'

describe('CommitItem', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Footer />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
