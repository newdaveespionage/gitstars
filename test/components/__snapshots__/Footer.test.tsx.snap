// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommitItem matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="jsx-3840341860"
  >
    <a
      class="jsx-3840341860"
      href="https://github.com/newdaveespionage/gitstars"
      rel="noopener noreferrer"
      target="_blank"
    >
      Created by Dave Espionage
    </a>
  </footer>
</DocumentFragment>
`;
